### PR TITLE
State requirement for Debian/Ubuntu based Docker images

### PR DESCRIPTION
**Pull request by csweichel**
in response to https://spectrum.chat/gitpod/general/custom-docker-image-ubuntu-debian-only~0498f5cf-2eb7-42c7-8373-c3945469e1bc, but should have been there all along.

### CODE REVIEW

File: src/docs/42_Config_Docker.md [hunk 1]
```diff
@@ -23,6 +23,8 @@ There are two ways to configure a custom Docker image in your `.gitpod.yml` file
 
 ## Creating Docker Images for Gitpod
 
+> Note: Gitpod only supports Debian/Ubuntu based images.
```
Comment (geropl) [inline src/docs/42_Config_Docker.md]: We talked about that, but have we decided yet to drop alpine?

Comment (32leaves) [reply src/docs/42_Config_Docker.md]: indeed we haven't. It felt wrong to "announce" alpine support here considering we might be dropping it soon. It's always easy to "add"/announce a feature compared to removing support for one.

Comment (32leaves) [reply src/docs/42_Config_Docker.md]: and Alpine isn't working at the moment

Comment (svenefftinge) [reply src/docs/42_Config_Docker.md]: Should be: `Currently, Gitpod only ...`

Comment (JesterOrNot) [reply src/docs/42_Config_Docker.md]: Mention that it is currently being worked on and possibly send a link to the issue so more people can pitch in

**Comment by JesterOrNot** [comment]
This was fixed in https://github.com/gitpod-io/website/pull/466

**Pull request closed without merging**